__tests__/data/singular.json

```diff
@@ -1700,9 +1704,6 @@
         "apns_token": "bk3RNwTe3H0CI2k_ HHwgIpoDKCIZvvD MExUdFQ3P1",
         "ua": "Dalvik/2.1.0 (Linux; U; Android 9; Android SDK built for x86 Build/PSR1.180720.075)",
         "attribution_token": "G9i5hC8lQJeGOfmS+MFycll/02...AAABBEQQBQS=",
-        "skan_conversion_value": "2",
-        "skan_first_call_timestamp": "1651853656",
-        "skan_last_call_timestamp": "1654532056",
```

Comment:
Why these are removed?

Reply:
The testcase was incorrectly written, for android `skan` values are not supported by Singular, seemingly ignored.